training/distributed_training/pytorch/data_parallel/yolov5/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training:1.10.2-gpu-py38-cu113-ubuntu20.04-sagemaker
```

Comment:
Can you fetch and use the customer's current region, rather than hard-coding us-east-1?

Reply:
Fixed, thanks!